xbmc/pvr/filesystem/PVRGUIDirectory.cpp

```diff
@@ -849,7 +849,7 @@ bool CPVRGUIDirectory::GetChannelsDirectory(CFileItemList& results) const
         if (dateAdded)
           item->SetProperty("hideable", true);
 
-        results.Add(std::move(item));
+        results.Add(item);
```

Comment:
Instead of this, please remove `const` in line 848.

Reply:
done